public/js/src/module/connection.js

```diff
@@ -653,14 +653,15 @@ function getDataFile(url, languageMap) {
         })
         .catch((error) => {
             const errorMsg =
-                error.message ||
-                t('error.dataloadfailed', {
-                    url,
-                    // switch off escaping just for this known safe value
-                    interpolation: {
-                        escapeValue: false,
-                    },
-                });
+                !error.message || /fetch/.test(error.message)
+                    ? t('error.dataloadfailed', {
+                          filename: url.replace(/.*\//, ''),
+                          // switch off escaping just for this known safe value
```

Comment:
Just a sanity check: are we sure this is known safe? I'd like to think so, since we're escaping these URLs. But turning off escaping here surprised me. Is that necessary or done for a particular reason?

Reply:
That's good to bring up. I think that was to display a full URL (and not sure if that was really safe...). I think that option can be removed. Will do so! Tx